packages/patternlab/styleguide/source/_patterns/03-organisms/by-template/breadcrumbs.twig

```diff
@@ -15,9 +15,18 @@
           </li>
         {% endif %}
       {% endfor %}
-      <li class="ma__breadcrumbs__item ma__breadcrumbs__item__expand-indicators" hidden="">
-        <button type="button" class="js-breadcrumbs-button" aria-label="Show all Breadcrumbs" title="Show all Breadcrumbs">…</button>
+      {% if breadcrumbs.currentPage %}
+         <li class="ma__breadcrumbs__item--current ma__visually-hidden" aria-current="location">{{ breadcrumbs.currentPage }}</li>
+      {% endif %}
+      <li class="ma__breadcrumbs__item ma__breadcrumbs__item__expand-indicators">
+        <button 
+          type="button"
+          class="js-breadcrumbs-button"
+          aria-label="Show all Breadcrumbs"
+          aria-describedby="button-description"
+        >…</button>
+        <div id="button-description" class="ma__visually-hidden">This page is located more than 3 levels deep within a topic. Some page levels are currently hidden. Use this button to show and access all levels.</div>
```

Comment:
Thank you so much for the all changes.

Please add `aria-hidden="true"` to `div#button-description`.  Without it, its content gets announced when screen readers are on the element.  The goal is to get it announced only when the button gets focus to deliver the information to screen reader users.